### DISCUSSION TITLE
Accidental release Symfony1 v1.15.16

### DISCUSSION THREAD

**Discussion by mentalstring** [FriendsOfSymfony1/symfony1#323] [General]
It seems that we have accidentally [fast-forwarded](https://github.com/FriendsOfSymfony1/symfony1/releases) symfony1 from v1.**5**.15 to v1.**15**.16!  I checked with @thePanz and it was just a typo 😅 – happens to the best.

It's marked as a _pre-release_ though – does anyone know wether that's still reversible? Can it simply be deleted? I don't have experience with maintaining releases, so I'm not sure what it entails. At the moment a `composer require friendsofsymfony1/symfony1` already installs `v1.15.16`.

**Comment by connorhu** [comment]
Nope, that is a release from the future, from 3024.

**Comment by connorhu** [comment]
@thePanz Can you rename the tag?
One of my repo:
![kép](https://github.com/FriendsOfSymfony1/symfony1/assets/1845575/4a9bc7b2-4719-4aa9-ac19-9bd6b1ce089d)

**Reply by thePanz** [reply]
I will try, worst case I'll delete and recreate the tag on the same commit

I expect users of SF1 to be super careful on updates, while keeping an eye to the repo (as for all other libraries). Hopefully this discussion will help to clarify the change 😀

**Comment by thePanz** [comment]
This is now fixed, deleted the wrong version and tagged `v1.5.16`

See: https://github.com/FriendsOfSymfony1/symfony1/releases/tag/v1.5.16

**Reply by connorhu** [reply]
I guess, you have to poke manually the packagist: https://packagist.org/packages/friendsofsymfony1/symfony1

**Reply by thePanz** [reply]
Done, release deleted there too!
Thanks for the heads up!

**Comment by thePanz** [comment]
Version stats:

Daily installs per version, averaged weekly for version 1.5
![image](https://github.com/FriendsOfSymfony1/symfony1/assets/226021/47365438-a45e-4641-999c-7635cade6b5a)

PHP version statistics:
![image](https://github.com/FriendsOfSymfony1/symfony1/assets/226021/5097f275-d096-4b4a-8b6f-23cb9c670020)

**Reply by mentalstring** [reply]
Usage is going... up? 🤔

**Reply by thirsch** [reply]
Maybe more and more are stumbling upon this repo and are happy to find an easy way to keep their sf1 running under newer php versions? Hopefully no new sf1 based projects are started. ;)

**Reply by connorhu** [reply]
Or just some CI process.

**Reply by thirsch** [reply]
Currently, we are using a fork as well. But as soon as we dropped all custom implementations or got them merged into here, we would contribute to the graph as well :)

**Reply by thePanz** [reply]
@thirsch yes, please keep the PRs coming

The SF1 has a lot of forks, and we should aim at unifying the maintenance work by havinf more people contributing here rather than spreading the efforts on hundred different forks :)

About the numbers going up: it might be as well the CI running and pulling the library more often

**Reply by connorhu** [reply]
I found global numbers: https://stitcher.io/blog/php-version-stats-january-2024
